gravitee-apim-portal-webui-next/src/app/registration/registration.component.html

```diff
@@ -0,0 +1,118 @@
+<!--
+
+    Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+@if (!submitted) {
+  <form
+    [formGroup]="registrationForm"
+    (ngSubmit)="register()"
+    class="registration__form__container"
+    [appIsMobile]="'registration__form__container--mobile'">
+    <mat-card appearance="outlined" class="registrationn">
+      <mat-card-header class="registration__form__header">
+        <mat-card-title class="registration__form__title" i18n="@@registrationTitle">CREATE AN ACCOUNT</mat-card-title>
```

Comment:
For consistency, could we use the same header casing as in the Login component?

Also, it's common to omit articles in headers for brevity and clarity

```suggestion
        <mat-card-title class="registration__form__title" i18n="@@registrationTitle">Create account</mat-card-title>